simple/README.md

```diff
@@ -1,17 +1,17 @@
-# Ratatui Simple template
+# Ratatui Simple Template
 
 The simple template will create the following project structure:
 
 ```text
 src/
 ├── app.rs     -> holds the state and application logic
-├── main.rs    -> entry-point
+└── main.rs    -> entry-point
 ```
 
 ## Design choices
 
 We have a small `App` struct that has a main loop that calls methods to handle events and draw the
-ui. The app can be quit by pressing any of Q/Esc/Ctrl+C.
+UI. The app can be quit by pressing any of Q/Esc/Ctrl+C.
 
-We use [color-eyre](https://docs.rs/color-eyre/latest/color_eyre/) for simplifying any errors that
+We use [color-eyre](https://docs.rs/color-eyre) for simplifying any errors that
 need to be reported to the console.
```

Comment:
nit: wrap at 100

Reply:
This seems already formatted to me. Can you maybe run your formatter and push the changes?